extensions/wikia/PremiumPageHeader/styles/_local-navigation.scss

```diff
@@ -0,0 +1,197 @@
+		cursor: pointer;
+		padding: $menu-item-vertical-padding $menu-item-horizontal-padding;
+		position: relative;
+		vertical-align: middle;
+
+		&:first-child {
+			margin-left: -$menu-item-horizontal-padding;
+		}
+
+		> a {
+			color: $color-button-text;
+			font-size: 12px;
+			font-weight: bold;
+			padding-right: 6px;
+			position: relative;
+			text-transform: uppercase;
+
+			.wds-icon {
+				height: 14px;
+				margin-right: 4px;
+				min-width: 14px;
+				vertical-align: top;
+				width: 14px;
+			}
+		}
+
+		&.pph-local-nav-container > a {
+			&:after, &:before {
+				border: solid transparent;
+				bottom: -$menu-item-vertical-padding - 1px;
+				content: '';
+				display: none;
+				height: 0;
+				left: 50%;
+				pointer-events: none;
+				position: absolute;
+				width: 0;
+			}
+
+			&:after {
+				border-bottom-color: $submenu-background-color;
+				border-width: $submenu-triangle-size;
+				margin-left: -$submenu-triangle-size;
+			}
+
+			&:before {
+				border-bottom-color: $submenu-border-color;
+				border-width: $submenu-triangle-size + 1px;
+				margin-left: -$submenu-triangle-size - 1px;
+			}
+		}
+
+		.pph-local-nav-chevron {
+			display: inline-block;
+			height: 8px;
+			min-width: 8px;
+			opacity: .5;
+			transition: opacity .1s;
+			vertical-align: baseline;
+			width: 8px;
+		}
+
+		&:hover {
+			a {
+				text-decoration: none;
+			}
+
+			.pph-local-nav-chevron {
+				opacity: 1;
+				transform: rotate(180deg);
+			}
+		}
+	}
+
+	li:hover {
+		> .pph-local-nav-sub-menu {
+			display: block;
+			z-index: 0;
+		}
+
+		&.pph-local-nav-container > a {
+			z-index: 1;
+
+			&:before, &:after {
+				display: block;
+			}
+		}
+	}
+}
+
+.pph-local-nav-sub-chevron {
+	transform: rotate(-90deg);
+}
+
+.pph-local-nav-sub-menu {
+	background: $submenu-background-color;
+	border: solid $submenu-border-color;
+	border-width: 0 1px 1px;
+	display: none;
+	min-width: $submenu-minimun-width;
+	padding: 12px 0 12px 12px;
+	position: absolute;
+
+	li {
+		align-items: center;
+		display: flex;
+		justify-content: space-between;
+		padding-right: 24px;
+		transition: background-color .2s;
+
+		a {
+			color: $color-text;
+			line-height: 14px;
+			padding: 12px 0 12px 12px;
+			transition: color .2s;
+			white-space: nowrap;
+			width: 100%;
+		}
+
+		.pph-local-nav-sub-chevron {
+			fill: $color-text;
+			margin-bottom: 2px;
+			margin-left: 8px;
+			transition: fill .2s;
+		}
+
+		&:hover {
+			background-color: rgba($color-links, .1);
+
+			> a {
+				color: $color-links;
+			}
+
+			.pph-local-nav-sub-chevron {
+				fill: $color-links;
+			}
+		}
+	}
+
+	&.pph-local-nav-l2 {
+		left: -24px + $menu-item-horizontal-padding;
+		margin-top: $menu-item-vertical-padding;
+
+		li a {
+			font-size: 14px;
+			font-weight: bold;
+		}
+	}
+
+	&.pph-local-nav-l3 {
+		box-sizing: border-box;
+		left: 100%;
+		min-width: 100%;
+		min-height: -moz-calc(100% + 1px);
+		min-height: -webkit-calc(100% + 1px);
+		min-height: calc(100% + 1px);
```

Comment:
All browsers that we support, support `calc` without prefix http://caniuse.com/#feat=calc